docs/tutorials/qml/ml_tools/callbacks.md

```diff
@@ -0,0 +1,231 @@
+
+# Callbacks for Trainer
+
+Qadence ml_tools provides a powerful callback system for customizing various stages of the training process. With callbacks, you can monitor, log, save, and alter your training workflow efficiently. A `CallbackManager` is used with [`Trainer`][qadence.ml_tools.Trainer] to execute the training process with defined callbacks. The default callbacks already provided in the [`Trainer`][qadence.ml_tools.Trainer].
+
+### Default Callbacks
+Below is a list of the default callbacks already implemented in the `CallbackManager` used with [`Trainer`][qadence.ml_tools.Trainer]:
+
+- **`on_train_start`**: `PlotMetrics`, `SaveCheckpoint`, `WriteMetrics`
```

Comment:
Maybe the `on` can be omitted ? Since it is the argument name, it feels a bit redundant.

Reply:
Updated.